<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatrixEmulator</class>
 <widget class="QWidget" name="MatrixEmulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>emu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="SchWidget" name="schWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Sch√∂nerz Dorm View</string>
      </attribute>
     </widget>
     <widget class="GridWidget" name="gridWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Grid View</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SchWidget</class>
   <extends>QWidget</extends>
   <header>schwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GridWidget</class>
   <extends>QWidget</extends>
   <header>gridwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
